### ISSUE TITLE
Regularization constant in Python implementation

### ISSUE THREAD

**Issue by andy920262** [cjlin1/simpleNN#5]
Hi, I find the regularization constant is different from where the README describe .
The README say `C=0.01` but it's set `math.inf` actually in the code.

![註解 2020-04-06 105111](https://user-images.githubusercontent.com/14959701/78519196-2a624e80-77f5-11ea-88b9-633b7fb779f7.png)

**Comment by quanpr** [comment]
Yes. You are right. Thanks for pointing that out. The Python code has been updated.
